coldfront/core/allocation/views.py

```diff
@@ -1716,6 +1716,22 @@ class AllocationClusterAccountRequestListView(LoginRequiredMixin,
     template_name = 'allocation/allocation_cluster_account_request_list.html'
     login_url = '/'
 
+    def get_queryset(self):
+        order_by = self.request.GET.get('order_by')
+        if order_by:
+            direction = self.request.GET.get('direction')
+            if direction == 'asc':
+                direction = ''
+            else:
+                direction = '-'
+            order_by = direction + order_by
+        else:
+            order_by = 'id'
+        cluster_account_status = AllocationAttributeType.objects.get(name='Cluster Account Status')
```

Comment:
Let's keep lines to 80 characters for readability's sake. (See the diff for how these two lines were broken up before.)